src/components/NcAppNavigationSettings/NcAppNavigationSettings.vue

```diff
@@ -27,6 +27,9 @@
 		<div id="app-settings__header">
 			<button class="settings-button"
 				type="button"
+				:aria-expanded="open"
```

Comment:
I am not entirely sure, but I think this needs to be a string, instead of a `Boolean`. At least we do it in every other place like this:
```suggestion
				:aria-expanded="open.toString()"
```

For example see:
https://github.com/nextcloud/nextcloud-vue/blob/b8e7742fce4024beeaf6c6b35fc32220c452ff2c/src/components/NcAppNavigationItem/NcAppNavigationItem.vue#L228
Sorry, that I didn't notice this earlier.

Reply:
Fixed